File: exercises/beecrowd/1007/1007.go
// https://judge.beecrowd.com/pt/problems/view/1007
package main

import "fmt"

func main() {
	var a int
	var b int
	var c int
	var d int

	fmt.Scanln(&a)
	fmt.Scanln(&b)
	fmt.Scanln(&c)
	fmt.Scanln(&d)

	fmt.Println("DIFERENCA =", a*b-c*d)
}

File: exercises/beecrowd/1007/1007.ts
// https://judge.beecrowd.com/pt/problems/view/1007
import * as fs from 'fs';

function main(): void {
    const input = fs.readFileSync('/dev/stdin', 'utf8');
    const lines = input.split(/\r?\n/);
    
    const a = Number(lines.shift());
    const b = Number(lines.shift());
    const c = Number(lines.shift());
    const d = Number(lines.shift());
    
    console.log(`DIFERENCA = ${a*b - c*d}`);
}

main()